### PR TITLE
Change first incentivized light protocol from Store to Lightpush

### PR DESCRIPTION
**Pull request by s-tikhomirov**
This PR changes the incentivization spec to outline Lightpush (instead of Store, as planned previously) to become the first incentivized light protocol in Waku.

### CODE REVIEW

File: standards/core/incentivization.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ In that case, a client MUST provide a valid eligibility proof as part of its req
 Forms of eligibility proofs include:
 
 - Proof of payment: for paid non-authenticated requests. A proof of payment, in turn, may also take different forms, such as a transaction hash or a ZK-proof. In order to interpret a proof of payment, the server needs information about its type.
-- Proof of membership: for services for a predefined group of users. An example use case: an application developer pays in bulk for their users' requests. A client then prove that they belong to the user set of that application. A similar concept (RLN) is used in Waku Relay for spam prevention.
+- Proof of membership: for services for a predefined group of users. An example use case: an application developer pays in bulk for their users' requests. A client then prove that they belong to the user set of that application. Rate limiting in Waku RLN Relay is based a similar concept.
```
Comment (jm-clius) [inline standards/core/incentivization.md:44]:
```suggestion
- Proof of membership: for services for a predefined group of users. An example use case: an application developer pays in bulk for their users' requests. A client then prove that they belong to the user set of that application. Rate limiting in Waku RLN Relay is based on a similar concept.
```

File: standards/core/incentivization.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ In an incentivized request-response protocol, only eligible (e.g., paying) clien
 Clients include eligibility proofs in their requests.
 
 Eligibility proofs are designed to be used in multiple Waku protocols, such as Store, Lightpush, and Filter.
-Store is planned to become the first Waku protocol to support incentivization.
-We discuss the proof-of-concept implementation of incentivization for Store in a later section.
+Lightpush is planned to become the first Waku protocol to support incentivization.
```
Comment (jm-clius) [inline standards/core/incentivization.md:18]: I think it's important to note that it's _attaching RLN proofs_ to lightpush requests that's the incentivised service here. It's perfectly possible for a client to generate and attach its own proofs and simply require the service to publish on its behalf, which is out of scope for now.

**Review by jm-clius — APPROVED** [review]
LGTM. Minor comments below.

File: standards/core/incentivization.md (hunk 2 above)
Comment (s-tikhomirov) [reply standards/core/incentivization.md]: Good point, I'll clarify that.

**Pull request merged**